ansible-scylla-node/defaults/main.yml

```diff
@@ -291,6 +291,36 @@ data_encryption_local:
   secret_key_block_mode: CBC # mandatory if you're not providing your own key
   secret_key_padding: PKCS5 # mandatory if you're not providing your own key
 
+# Encrypt system resources:
+# System encryption is applied to semi-transient on-disk data, such as commit logs, batch logs, and hinted handoff data.
+# If you'd like to use a LocalKeyProvider, please use the system_info_encrypt_local config
+# If you'd like to use a KMIPKeyProvider, please use the system_info_encryption_kmip config
+
+# LocalKeyProvider:
+# To use your own key, copy it to {{ localhost_key_dir }} with the name {{ key_name }} and then this key will be copied to all the nodes.
```

Comment:
And what if one wants to have a separate key for every node? 
Scylla supports it just fine AFAIU.

Reply:
You're right. This is supported by this PR now.